### PR TITLE
feat: create workflow for current examples

### LINKED ISSUE #2473: create a scheduled workflow that will run all of the `examples`
To ensure that our examples stay up to date and don't break, we should run them in our pipeline.  However, we don't want to run them every push for a PR, as that will slow down our PRs, and they are already taking to long.  Instead we schedule a batch job to run daily during hours where Solo is least active.  It should use a matrix JSON and run for each example in the example folder.  Our existing `.github/workflows/zxc-e2e-test.yaml` should work with minor adjustments, and just have a new workflow that leverages it.

Having this before we tag the release with the command renames would be beneficial to increase probability that our examples match all of the code changes.

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

* add new workflow to run examples
* based on input variable to use latest solo or release solo to run test
* lower resource request in `examples/custom-network-config/init-containers-values.yaml` so the test can run on local PC
* also fix the bug that port forward configuration not saved in remote config

### Test result

https://github.com/hiero-ledger/solo/actions/runs/17142132617

### Related Issues

* Closes #2473

### Pull request (PR) checklist

* \[ ] This PR added tests (unit, integration, and/or end-to-end)
* \[ ] This PR updated documentation
* \[ ] This PR added no TODOs or commented out code
* \[ ] This PR has no breaking changes
* \[ ] Any technical debt has been documented as a separate issue and linked to this PR
* \[ ] Any `package.json` changes have been explained to and approved by a repository manager
* \[ ] All related issues have been linked to this PR
* \[ ] All changes in this PR are included in the description
* \[ ] When this PR merges the commits will be squashed and the title will be used as the commit message, the 'commit message guidelines' below have been followed

### Testing

* \[ ] This PR added unit tests
* \[ ] This PR added integration/end-to-end tests
* \[ ] These changes required manual testing that is documented below
* \[ ] Anything not tested is documented

The following manual testing was done:

* TBD

The following was not tested:

* TBD

<details>
<summary>
Commit message guidelines
</summary>
We use 'Conventional Commits' to ensure that our commit messages are easy to read, follow a consistent format, and for automated release note generation. Please follow the guidelines below when writing your commit messages:

1. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.  NOTE: currently breaking changes will only bump the MAJOR version.
2. The title is prefixed with one of the following:

| Prefix    | Description                                         | Semantic Version Update | Captured in Release Notes |
|-----------|-----------------------------------------------------|-------------------------|---------------------------|
| feat:     | a new feature                                       | MINOR                   | Yes                       |
| fix:      | a bug fix                                           | PATCH                   | Yes                       |
| perf:     | performance                                         | PATCH                   | Yes                       |
| refactor: | code change that isn't feature or fix               | none                    | No                        |
| test:     | adding missing tests                                | none                    | No                        |
| docs:     | changes to documentation                            | none                    | Yes                        |
| build:    | changes to build process                            | none                    | No                        |
| ci:       | changes to CI configuration                         | none                    | No                        |
| style:    | formatting, missing semi-colons, etc                | none                    | No                        |
| chore:    | updating grunt tasks etc; no production code change | none                    | No                        |

</details>

### CODE REVIEW

File: examples/custom-network-config/Taskfile.yml [hunk 1]
```diff
@@ -1,4 +1,14 @@
 version: 3
+
+env:
+  SOLO_COMMAND:
+    sh: |
+      if [ "${USE_RELEASED_VERSION}" = "true" ]; then
+        echo "npx solo"
```
Comment (nathanklick) [inline examples/custom-network-config/Taskfile.yml:7]:
```suggestion
        echo "npx @hashgraph/solo"
```

**Review by instamenta — APPROVED** [review]
